### PR TITLE
Added sound effects

### PR DESCRIPTION
**Pull request by bvandenbogaard**
- Voices for most units
- Sell sounds
- Fixed EVA
- Structure sounds
- Added new notifications
- Added building damaged sounds
- Unit repaired notification
- Added new destroyed sounds for power plants

### CODE REVIEW

File: rules/defaults.yaml [hunk 1]
```diff
@@ -103,7 +103,7 @@
 	Building:
 		Dimensions: 1,1
 		Footprint: x
-		BuildSounds: uplace.wav
+		BuildSounds: uplace.wav
```
Comment (abcdefg30) [inline rules/defaults.yaml]: Please undo this change.

**Comment by abcdefg30** [comment]
Looks good to me. :+1: / :white_check_mark: Thanks!

File: rules/structures.yaml [hunk 2]
```diff
@@ -811,6 +815,8 @@ nanrct:
 	Power:
 		Amount: 2000
 	ScalePowerWithHealth:
+    SoundOnDamageTransition:
```
Comment (Mailaender) [inline rules/structures.yaml]: Broken indention.

Comment (Phrohdoh) [inline rules/structures.yaml]: This was fixed in a later commit, iirc.
